content/v2.0/reference/release-notes/influxdb.md

```diff
@@ -8,6 +8,28 @@ menu:
 weight: 101
 ---
 
+## v2.0.0-beta.12 [2020-06-12]
+
+### Features
+
+- Make all `pkg` applications stateful through stacks.
+- Add ability to export a stack's existing resource state as a `pkg`.
+- Update influx `pkg` commands with improved usage and examples in long form.
+- Extend `influx` CLI with version and User-Agent.
+- Integrate `RedirectTo` functionality so Cloud users are redirected back to the originally linked page after login.
+- Consolidate `pkg` influx commands under templates. Remove some nesting of the `influx` CLI commands.
```

Comment:
were all `pkg` commands moved to influx apply?

Reply:
some were, but not all, we have the following additions (previous commands will still work):

```
influx pkg => influx apply
influx pkg export => influx export
influx pkg stack => influx stacks
influx pkg summarize|validate => `influx template` and `influx template validate`
```